tests/State/ScopeTest.php

```diff
@@ -122,7 +113,7 @@ public function testAddEventProcessor(): void
         $this->assertSame($event, $scope->applyToEvent($event, []));
         $this->assertTrue($callback1Called);
 
-        $scope->addEventProcessor(function () use ($callback1Called, &$callback2Called, $callback3Called): ?Event {
+        $scope->addEventProcessor(function () use ($callback1Called, &$callback2Called, $callback3Called) {
```

Comment:
why did you remove `?Event` return value?

Reply:
I said to you in another PR to add them but rethinking about it it makes no sense if the only return type of the lambda is an hardcoded `null` value